executor/runtime/docker/docker.go

```diff
@@ -460,7 +460,8 @@ func (r *DockerRuntime) mainContainerDockerConfig(c runtimeTypes.Container, bind
 
 	// Always setup tmpfs: it's needed to ensure Metatron credentials don't persist across reboots and for SystemD to work
 	hostCfg.Tmpfs = map[string]string{
-		"/run": "rw,exec,size=" + defaultRunTmpFsSize,
+		"/run":       "rw,exec,size=" + defaultRunTmpFsSize,
+		"/run/netns": "rw,size=" + defaultRunTmpFsSize,
```

Comment:
Do we really need this to be *in* the container? Why can't it live in /var/run/titus-executor/ something?

Reply:
Just adds to the surface area of what is inside the container and changes the behavior on all containers (even non-transition ones).

Reply:
I mean, I'm trying to make it so that standard tools (like `ip netns`) will work with this. They expect /run/netns to be a tmpfs. I can put this behind a flag so it's only done on containers with transition mode?

Reply:
Oh. Huh. 
No this is fine I guess. As long as users don't depend on this.

I still think it is a little weird that we end up mounting this ns ... directory (?) *in* the container, only to titus-mnt it *back* out for titus-vpc-tool to run? (am I understanding it correctly?)

Reply:
it's a file. So, the transition namespace is /run/netns/transition. This is also the way the TSA figures out which transition namespace to use for a given container.